fuel-tests/tests/tx/utxo_validation.rs

```diff
@@ -172,9 +173,9 @@ async fn dry_run_override_utxo_validation() {
     assert_eq!(2, log.len());
 
     assert!(matches!(log[0],
-        Receipt::Return {
-            val, ..
-        } if val == 1));
+        Receipt::Panic {
```

Comment:
The test should be positive, you need to increase the gas limit

Reply:
fixed!